src/game.cpp

```diff
@@ -1522,7 +1522,54 @@ bool game::do_turn()
     }
     sfx::do_danger_music();
     sfx::do_fatigue();
+#ifdef GSI
```

Comment:
This block of code needs to move to a gsi file, it looks like it just needs a reference to the player object and a few other things to access the data it needs.

Along with this the ifdef would be on the gsi file.

Reply:
Thanks for the input.  I'm working on moving stuff over to a minimal amount of GSI methods and getting rid of the ifdefs in main files.  

Multithreading stuff should be gone at this point, and I believe I should be able to do everything I want to do with sockets without relying on multithreading.

The gamestate.json file shouldnt be an issue since I am no longer using file I/O, I'll delete it by the time this is ready for merging, but there's still a lot of work to be done.